packages/react-charting/src/components/LineChart/__snapshots__/LineChart.test.tsx.snap

```diff
@@ -45,7 +45,7 @@ exports[`LineChart - mouse events Should render callout correctly on mouseover 1
           "display": "block",
         }
       }
-      width={100}
+      width={108}
```

Comment:
Dunno what is happening here but this looks like not the most robust approach to approach tests. 

It would be great to change this long term. also there are dozens of errors being triggered because react-charting is not using `act` as needed within tests. 

cc @AtishayMsft  - if you're ok with these changes lets merge it and unblock everyone else. your team can provide proper fixes afterwards. WDYT ?

Reply:
Agreed. Upgrading jest version should not break test snapshots. 
We have encountered some issues with the test platform in the past and had to workaround those.
Is this the only test that failed?
We can file a bug and let this upgrade proceed. The act issue is on our radar but not prioritized yet.

@srmukher Could you log a bug for this on the charting side.